endpointmanager/pkg/endpointlinker/endpointlinker.go

```diff
@@ -100,7 +102,7 @@ func getIdsOfMatchingNPIOrgs(npiOrgNames []*endpointmanager.NPIOrganization, nor
 		}
 		if consideredMatch {
 			// multiply confidence by .9 for all name matches to demonstrate that these matches are not as good as the id matches
```

Comment:
Just nit picking here- update the comment to .99

Reply:
Thanks for pointing this out, just fixed it!